branded_ios_app/publishing_ios_app_7.rst

```diff
@@ -123,6 +124,16 @@ iPhoneOSProgrammingGuide/BackgroundExecution/BackgroundExecution.html>`_
 
 .. figure:: images/ownbrander-32.png
 
+ownBrander auto-generates a version number for you. You have the option to 
+customize your version number. Once you do this, you will have to update it 
+manually in new releases. You may also modify the build number, which defaults 
+to 1.0. This must also be manually updated. The version number must exactly 
+match the version number you enter in iTunes. Your build number is used by 
+iTunes to uniquely identify your app. When the build number changes, iTunes 
+automatically syncs the updates for your users.
+
```

Comment:
I'm afraid that this paragraph doesn't appear to me very clear :unamused: 
I'd distinguish in 2 paragraph version number and build number, indeed they may be modified independenly

- Version number: this is the number which is shown to the end user, in our case, 3.4.8.  Once, this is modified by the customer, it has to be increased everytime a new version is released. It has to be the same one that it is filled in iTunes when preparing a new version

- Build number, this is an internal number unvisible for the end user, by default it's set to 1.0.0. What is the aim of this one? Imagine you have sent a new version for Apple to be approved (this process takes about 1 week), suddenly, you realise that there is a typo on the app, instead of having to change the version number, you may change the build number; so that you may send a v2 to Apple without having to increase the version number (the one visible to your end users). There is no need to increase this number between releases because the version number will be increased